website/docs/guides/orchestration/webhooks/zapier-new-cloud-job.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Trigger another dbt Cloud job when a run finishes"
+id: webhooks-guide-zapier-new-cloud-job
+slug: zapier-new-cloud-job
+---
+
+This guide assumes familiarity with:
+- dbt Cloud webhooks
+- Zapier
+
+## What does this integration do? 
+When a dbt Cloud job completes:
```

Comment:
similar feedback to MS teams integration; either list these or add commas and ands